apps/dashboard/src/api/activity.ts

```diff
@@ -0,0 +1,51 @@
+import { IActivity, IEnvironment } from '@novu/shared';
+import { get } from './api.client';
+
+export interface IActivityFilters {
+  channels?: string[];
+  templates?: string[];
+  email?: string;
+  subscriberId?: string;
+  transactionId?: string;
+}
+
+interface ActivityResponse {
+  data: IActivity[];
+  hasMore: boolean;
+  pageSize: number;
+}
+
+export function getActivityList(
+  environment: IEnvironment,
+  page = 0,
+  filters?: IActivityFilters
+): Promise<ActivityResponse> {
+  const searchParams = new URLSearchParams();
+  searchParams.append('page', page.toString());
```

Comment:
Does the API work with page or limit offset? I see both in different parts of the code.

Reply:
It uses a page, I will adjust it in the other prs 💪